src/lib/viewers/doc/BoxExcel/sheet.js

```diff
@@ -0,0 +1,640 @@
+            }
+        }
+        if (sheet['!cols']) {
+            for (let i = 0; i < columnCount - 1; ++i) {
+                col = sheet['!cols'][i];
+                if (col && col.wpx) columnWidths = columnWidths.set(i, col.wpx);
+                if (col && col.hidden) columnWidths = columnWidths.set(i, 0);
+            }
+        }
+
+        const merges = {};
+        let totalHeight;
+        let totalWidth;
+        if (sheet['!merges']) {
+            sheet['!merges'].forEach((merge) => {
+                totalHeight = 0;
+                totalWidth = 0;
+                for (let rowIndex = merge.s.r; rowIndex <= merge.e.r; ++rowIndex) {
+                    for (let colIndex = merge.s.c; colIndex <= merge.e.c; ++colIndex) {
+                        merges[[rowIndex, colIndex]] = { display: 'none' };
+                    }
+                    totalHeight += rowHeights.get(rowIndex, rowHeight);
+                }
+                for (let colIndex = merge.s.c; colIndex <= merge.e.c; ++colIndex) {
+                    totalWidth += columnWidths.get(colIndex, columnWidth);
+                }
+                merges[[merge.s.r, merge.s.c]] = {
+                    height: totalHeight,
+                    width: totalWidth
+                };
+            });
+        }
+
+        return {
+            rowCount,
+            columnCount,
+            rowHeight,
+            columnWidth,
+            rowHeights,
+            columnWidths,
+            merges
+        };
+    };
+
+    /**
+     * Get data object for a certain position
+     *
+     * @param  {number} rowIndex    row index, 1 basis
+     * @param  {number} columnIndex col index, 1 basis
+     * @return {Object}             data object for the cell
+     */
+    _getCell = (rowIndex, columnIndex) => {
+        /* global XLSX */
+        const pos = XLSX.utils.encode_cell({
+            r: rowIndex - 1,
+            c: columnIndex - 1
+        });
+        const { sheet } = this.state;
+        return sheet[pos];
+    };
+
+    /**
+     * Helper function of _getBorder
+     *
+     * @param  {Object} style style property in a cell object
+     * @return {string}       hex color string
+     */
+    _getBorderColor = (style) => {
+        return style && style.color ? _parseColor(style.color) : c.gridGrey;
+    };
+
+    /**
+     * Helper function of _getBorder
+     *
+     * @param  {style} style style property in a cell object
+     * @return {string}      border width string, default: '1px'
+     */
+    _getBorderWidth = (style) => {
+        return style && style.style && borderWidthMap[style.style] ? borderWidthMap[style.style] : '1px';
+    };
+
+    /**
+     * Get border style, including borderColor and borderWidth
+     *
+     * @param  {Object} style style property in a cell object
+     * @return {Object}       React style object
+     */
+    _getBorder = (style) => {
+        const borderColor = `${this._getBorderColor(style.top)} ${this._getBorderColor(
```

Comment:
move each of these border widths to the top of this method that way borderWidth will just be `${top} ${right}....`
i.e.

`const topWidth = style.top ? this.getBorderWidth(style.top) : '0px'`;